cmd/eksctl/main.go

```diff
@@ -24,11 +24,12 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
-
 	addCommands()
 
 	rootCmd.PersistentFlags().IntVarP(&logger.Level, "verbose", "v", 3, "set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging")
 	rootCmd.PersistentFlags().BoolVarP(&logger.Color, "color", "C", true, "toggle colorized logs")
```

Comment:
I would rather change the `--color` flag to take a string, we can easily keep it compatible with current behaviour by supporting `--color=false`.